### PR TITLE
NN-1439 Add Date of Birth filter

### PR DESCRIPTION
**Pull request by bell-pa**

### CODE REVIEW

File: src/GlobalSearch/DateOfBirth.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    }
+    this.onChange = this.onChange.bind(this)
+    this.errorText = this.errorText.bind(this)
+  }
+
+  componentDidMount() {
+    this.onChange({})
+  }
+
+  onChange(value) {
```
Comment (jsrobertson) [inline src/GlobalSearch/DateOfBirth.js]:
```suggestion
  onChange = value => {
```

Comment (jsrobertson) [reply src/GlobalSearch/DateOfBirth.js]: Not a deal breaker but using class properties like ^ means you don't have to bind it in the constructor.

File: package.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@govuk-react/constants": "^0.3.1",
     "@govuk-react/header": "^0.3.1",
+    "@govuk-react/date-field": "^0.3.1",
```
Comment (jsrobertson) [inline package.json]: 👍

File: src/GlobalSearch/DateOfBirth.js (hunk 1 above)
Comment (bell-pa) [reply src/GlobalSearch/DateOfBirth.js]: Done

**Pull request merged**